sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/__init__.py

```diff
@@ -2,4 +2,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+from ._access_control_client import KeyVaultAccessControlClient
+from ._internal.client_base import ApiVersion
+from ._models import KeyVaultPermission, RoleAssignment, RoleDefinition
+
+
+__all__ = ["ApiVersion", "KeyVaultAccessControlClient", "KeyVaultPermission", "RoleAssignment", "RoleDefinition"]
```

Comment:
Regrading `ApiVersion`, is this the standard in Python? In JavaScript we are standardizing on `serviceVersion`.

Reply:
Yes, it's consistent with the Python SDK.